interceptors_logger.go

```diff
@@ -155,6 +155,9 @@ var emptyValRX = regexp.MustCompile(
 // requests and responses.
 func rprintReqOrRep(w io.Writer, obj interface{}) {
 	rv := reflect.ValueOf(obj).Elem()
+	if !rv.IsValid() {
```

Comment:
Huh. So the following wasn't sufficient?

```
	// Print the response data if it is set.
	if rep != nil {
		rprintReqOrRep(w, rep)
	}
```

Reply:
Hmm... now that is weird. I wonder if there is an unguarded call to this somewhere.. lemme check.